src/gpl/test/regression_tests.tcl

```diff
@@ -7,6 +7,7 @@ record_tests {
   simple01-ref
   simple01-skip-io
   simple01-rd
+  simple02-rd  
```

Comment:
Could you add it to CMakeLists.txt as well. We're moving away from this runner to CTest